dvb/cx23885/pkg.yaml

```diff
@@ -46,6 +46,9 @@ steps:
         cp -r /rootfs/ /extensions-validator-rootfs/rootfs
         cp /pkg/manifest.yaml /extensions-validator-rootfs/manifest.yaml
         /extensions-validator validate --rootfs=/extensions-validator-rootfs --pkg-name="${PKG_NAME}"
+    sbom:
+      outputPath: /rootfs/usr/local/share/spdx/cx23885.spdx.json
+      version: {{ .LINUX_DVB_FIRMWARE }}
```

Comment:
These all seem to be part of Linux + part of linux-firmware. We track Linux as a whole in the core Talos